it/utils/ViewHelpers.scala

```diff
@@ -219,13 +220,8 @@ trait ViewHelpers {
     }
   }
 
-  def inputFieldValueCheck(name: String, selector: String, value: String)(implicit document: () => Document): Unit = {
-    s"'$selector' has a name of '$name'" in {
-      document().select(selector).attr("name") shouldBe name
-    }
-    s"'$selector' has a value of '$value'" in {
-      document().select(selector).attr("value") shouldBe value
-    }
+  def getInputFieldValue(selector: String = "#amount")(implicit result: WSResponse): String = {
```

Comment:
Is there a reason that you supplemented the inputFieldValueCheck method above? This class contains Checks methods. Can you please change the method to make it in the same stile?

Reply:
Yes there is the testing is simpler this way.